docs/api-reference/uiSchema.md

```diff
@@ -71,6 +71,13 @@ Specify either the name of a field that is used to look up an implementation fro
 
 See [Custom Widgets and Fields](https://react-jsonschema-form.readthedocs.io/en/stable/api-reference/custom-widgets-fields#custom-field-components) for more information about how to use this property.
 
+### ui:fieldReplacesAnyOrOneOf
+
+By default, any field that is rendered for an `anyOf`/`oneOf` schema will be wrapped inside the `AnyOfField` or `OneOfField` component.
+By providing a `true` value for this flag in association with a custom `ui:field` the default behavior is skipped.
+Instead, it will be up to the custom field implementation to deal with rendering the `anyOf`/`oneOf` behavior.
+Providing a `false` value, will maintain the wrapping behavior rather than the replacing behavior.
```

Comment:
@nickgros / @epicfaace, This sounds a bit awkward even to me, so feel free to improve it if you can.